db/migrate/20160114121308_remove_link_fields.rb

```diff
@@ -0,0 +1,18 @@
+class RemoveLinkFields < ActiveRecord::Migration
```

Comment:
I think we should do this migration in a separate PR, but add in this PR a script which copies the data from these link fields into publishing-API.  Then we can deploy this PR, copy the data across to publishing API, and later remove the old data once everything is working nicely.